rmrk-token/src/lib.rs

```diff
@@ -14,36 +14,20 @@ use messages::*;
 mod mint;
 mod multiresource;
 use multiresource::*;
-
-gstd::metadata! {
-    title: "RMRK",
-    init:
-        input: InitRMRK,
-    handle:
-        input: RMRKAction,
-        output: RMRKEvent,
-    state:
-        input: RMRKState,
-        output: RMRKStateReply,
-}
-
-#[derive(Debug, Default, PartialEq, Eq)]
-struct RMRKOwner {
-    token_id: Option<TokenId>,
-    owner_id: ActorId,
-}
+mod utils;
+use hashbrown::HashMap;
 
 #[derive(Debug, Default)]
 struct RMRKToken {
     name: String,
     symbol: String,
     admin: ActorId,
-    token_approvals: BTreeMap<TokenId, BTreeSet<ActorId>>,
-    rmrk_owners: BTreeMap<TokenId, RMRKOwner>,
-    pending_children: BTreeMap<TokenId, BTreeSet<CollectionAndToken>>,
-    accepted_children: BTreeMap<TokenId, BTreeSet<CollectionAndToken>>,
-    children_status: BTreeMap<CollectionAndToken, ChildStatus>,
-    balances: BTreeMap<ActorId, TokenId>,
+    token_approvals: HashMap<TokenId, BTreeSet<ActorId>>,
+    rmrk_owners: HashMap<TokenId, RMRKOwner>,
+    pending_children: HashMap<TokenId, BTreeSet<CollectionAndToken>>,
+    accepted_children: HashMap<TokenId, BTreeSet<CollectionAndToken>>,
+    children_status: HashMap<CollectionAndToken, ChildStatus>,
+    balances: HashMap<ActorId, TokenId>,
     multiresource: MultiResource,
     resource_id: ActorId,
     equipped_tokens: BTreeSet<TokenId>,
```

Comment:
Your forget replace here like me in PR #20 
After this merge we should close #20

Reply:
Ok